### ISSUE TITLE
SCSS loader issue

### ISSUE THREAD

**Issue by Blueblau** [egoist/unvue#15]
Hi,

Tried the [pre-fetch-cache](https://github.com/egoist/unvue/tree/dev/examples/pre-fetch-cache) example.

I installed the `node-sass` and `sass-loader` to enable SCSS, but when I tried to add `lang="scss"` on the `style` block, I got this error in the terminal:

```bash
ERROR in ./src/views/Home.vue
Module not found: Error: Can't resolve 'scss-loader' in '/Users/kenneth/Projects/unvue-test/src/views'
 @ ./src/views/Home.vue 3:0-263
 @ ./src/router.js
 @ ./src/index.js
 @ ./~/unvue/app/client-entry.js
 @ multi ./~/unvue/app/dev-client.js @owndir/app/client-entry
```

**Comment by cannap** [comment]
i have the same but you can say lang="sass" and just write scss

**Comment by egoist** [comment]
Thanks for reporting this issue, it's fixed in `v0.9.7`
